cmd/kms-rest/startcmd/lddocument_loader.go

```diff
@@ -140,6 +205,10 @@ func loadLDContext() (map[string]*ld.RemoteDocument, error) {
 			vocab:   "https://w3id.org/security/v2",
 			content: w3idOrgSecurityV2,
 		},
+		{
+			vocab:   "https://trustbloc.github.io/context/vc/credentials-v1.jsonld",
```

Comment:
Where is this context referenced? Is it from aries-framework-go while adding `JsonWebSignature2020` proofs?

Reply:
I think so, I got it from tracing network calls from loader, also this is what is listed [here](https://github.com/hyperledger/aries-framework-go/blob/main/pkg/doc/signature/proof/cache.go#L9)

Reply:
I was going to say that aries-framework-go is _supposed_ to already be adding this context - apparently that is not working properly.

cc @kdimak